### DISCUSSION TITLE
next button

### DISCUSSION THREAD

**Discussion by dlaflamme1** [liedekef/events-made-easy#88] [Q&A] [answered]
Hello I would like to know how we can put like 9 event by page and hate a next button at the end of the page to go see more events
thank you

**Comment by liedekef** [comment, marked as answer]
Use paging=1 and limit=9 as options to the eme_events shortcode

**Comment by dlaflamme1** [comment]
Ok and it is possible to have like 9 on the desktop and 8 on mobile?
It is possible to hide the next page button of the top but keep the one at the bottom? And it is possible to stylize it and change the name for suivant ?
Thank you

De : Franky Van Liedekerke ***@***.***
Envoyé : 8 mars 2023 13:39
À : liedekef/events-made-easy ***@***.***>
Cc : David Laflamme ***@***.***>; Author ***@***.***>
Objet : Re: [liedekef/events-made-easy] next button (Discussion #88)


Use paging=1 and limit=9 as options to the eme_events shortcode

**Comment by liedekef** [comment]
Using the template_id_header, template_id_footer and the per-event template_id options, you can create a responsive layout of your own choosing. The 8 and/or 9 is then not relevant.
The next/previous buttons have classes eme_nav_right and eme_nav_left, so using css you can hide the first occurence of those and change the look and feel. Changing the text can be done by using some jquery to change the content (sorry, no other method here).
